samples/sensor/sgp40_sht4x/src/main.c

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+#include <usb/usb_device.h>
+
+/*
+ * User configurable options
+ */
+#define MEASURE_WITH_COMPENSATION 0
+#define USE_HEATER 1
```

Comment:
can you add Kconfig options to the sample?

Reply:
i made Kconfig options out of these